lcm/lcm_trajectory.cc

```diff
@@ -68,9 +79,9 @@ lcmt_saved_traj LcmTrajectory::generateLcmObject() const {
   traj.num_trajectories = trajectories_.size();
 
   // For each trajectory
-  for (auto & traj_el : trajectories_) {
+  for (auto& traj_el : trajectories_) {
     lcmt_trajectory_block traj_block;
-    const Trajectory* cpp_traj = &traj_el.second;
+    const Trajectory *cpp_traj = &traj_el.second;
```

Comment:
Per GSG, this should be `Trajectory* cpp_traj` not `Trajectory *cpp_traj` (the way your earlier commit had it)